website/scripts/convert_ipynb_to_mdx.py

```diff
@@ -347,7 +356,7 @@ def transform_code_cell(  # noqa: C901 (flake8 too complex)
                             # markdown.
                             md = ""
                             if isinstance(md_df.index, pd.RangeIndex):
-                                md = md_df.to_markdown(showindex=False)
+                                md = md_df.to_markdown(index=False)
```

Comment:
Our internal `pandas` and `tabulate` packages are a bit old so they don't support the `index` argument :(

Reply:
got it, will revert back